### PR TITLE
Update setValidTick to be an arrow function

### PR DESCRIPTION
**Pull request by drishit96**
## What does this PR do?
Update setValidTick to be an arrow function

### Associated Issue
Fixes (#3511 )

### Code
 - [x] Build Component in HTML
 - [ ] Build Component in Angular
 - [ ] Build Component in React
 - [ ] Unit Testing in HTML with `npm run test` in `html/` (100% coverage, 100% passing)
 - [ ] Unit Testing in Angular with `npm run test` in `angular/` (100% coverage, 100% passing)
 - [ ] Unit Testing in React with `npm run test` in `react/` (100% coverage, 100% passing)

### CODE REVIEW

File: html/utilities/validation/setValidTick.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function setValidTick(inputContainer) {
+const setValidTick = (inputContainer) => {
```
Comment (afebbraro) [inline html/utilities/validation/setValidTick.js:1]: hey @drishit96  thanks for contributing! Could you add a `;` to the end of this arrow function please? Thanks again!

**Review by afebbraro — CHANGES_REQUESTED** [review]

File: html/utilities/validation/setValidTick.js (hunk 1 above)
Comment (drishit96) [reply html/utilities/validation/setValidTick.js:1]: Done

**Review by drishit96 — COMMENTED** [review]
Added missing ;

**Review by guern1kn — APPROVED** [review]
Thanks for contributing!

**Pull request merged**